### PR TITLE
Fix mac-build issue in io::IoStatistics

### PR DESCRIPTION
**Pull request by vermapratyush**

### CODE REVIEW

File: velox/common/io/CMakeLists.txt [hunk 1]
```diff
@@ -11,4 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-add_library(velox_common_io IoStatistics.cpp)
+
+add_library(velox_common_io IoStatistics.h IoStatistics.cpp)
+
+target_link_libraries(velox_common_io Folly::folly ${FOLLY_BENCHMARK} glog::glog)
```
Comment (majetideepak) [inline velox/common/io/CMakeLists.txt:17]: should not need `${FOLLY_BENCHMARK}`

**Comment by vermapratyush** [comment]
@majetideepak After the change the build still failed: https://github.com/facebookincubator/velox/actions/runs/6434273760/job/17473129847?pr=6939

Do you know why?

I am now trying without the build cache.

**Comment by kgpai** [comment]
FYI, Then benchmark build uses merge base 'HEAD is now at 3ac11b2ac Use simpler PlanBuilder::finalAggregation API in TpchQueryBuilder (#6936)' for the build .

**Comment by kgpai** [comment]
The reason #6936 breaks is because its rebased on top of #6847 which was already breaking : https://github.com/facebookincubator/velox/actions/runs/6429713104/job/17459308863 

So theres nothing you can do in this PR to fix ubuntu benchmarks as that will refer to a broken historical point.

File: .github/workflows/benchmark.yml [hunk 2]
```diff
@@ -83,8 +83,7 @@ jobs:
           git remote add upstream https://github.com/facebookincubator/velox
           git fetch upstream
           git status
-          merge_base=$(git merge-base 'upstream/${{ github.base_ref }}' 'origin/${{ github.head_ref }}') || \
-            { echo "::error::Failed to find merge base"; exit 1; }
+          merge_base="d51190039648bc1b5725e0261889d19ea2e65a85"
```
Comment (kgpai) [inline .github/workflows/benchmark.yml:86]: We are using a last known good merge base to validate the build works and compare results against. 
After this lands we can revert to original code to determine merge base.

**Pull request closed without merging**